e2e/test/scenarios/dashboard-cards/dashboard-drill.cy.spec.js

```diff
@@ -645,6 +645,41 @@ describe("scenarios > dashboard > dashboard drill", () => {
     });
   });
 
+  it("should keep card's display when doing zoom drill-through from dashboard (metabase#38307)", () => {
+    cy.log("Create a question");
+
+    cy.createQuestion({
+      name: "38307",
+      query: {
+        "source-table": REVIEWS_ID,
+        aggregation: [["count"]],
+        breakout: [["field", REVIEWS.CREATED_AT, { "temporal-unit": "month" }]],
+      },
+      display: "bar",
+    }).then(({ body: { id: QUESTION_ID } }) => {
+      cy.createDashboard().then(({ body: { id: DASHBOARD_ID } }) => {
+        cy.log("Add question to the dashboard");
+        addOrUpdateDashboardCard({
+          card_id: QUESTION_ID,
+          dashboard_id: DASHBOARD_ID,
+        });
```

Comment:
We can use [`cy.createQuestionAndDashboard` command](https://github.com/metabase/metabase/blob/master/e2e/support/commands/api/composite/createQuestionAndDashboard.js) to do all that

Reply:
Ah, thanks for pointing that out :)